clients/src/main/java/org/apache/kafka/server/quota/ClientQuotaEntity.java

```diff
@@ -1,4 +1,5 @@
 /*
```

Comment:
RSPEC-2176: Class has same name as parent
<!-- lookout footnote separator -->
_Comment made by the analyzer sonarcheck_analyzer._